### DISCUSSION TITLE
Save your settings, conversations and everything

### DISCUSSION THREAD

**Discussion by Root-FTW** [AstraSurge/gpteams#48] [Ideas]
When I log into my google account on my work computer and use GPTeams, then when I get home at night and log into another PC with my same account all my conversations, settings and everything is not there, is it possible that everything is saved in my account?

**Comment by assassinliujie** [comment]
> 当我在工作计算机上登录我的谷歌帐户并使用 GPTeams 时，当我晚上回到家并使用我的同一帐户登录另一台 PC 时，我所有的对话、设置和所有内容都不在那里，是否有可能所有内容都保存在我的帐户中？

this may need a database

**Comment by suikodev** [comment]
This is a feature we will implement in version 1.1.0
